docs/release-notes/1.4.0-preview/release-1.4.0-preview.md

```diff
@@ -0,0 +1,88 @@
+# [ML.NET](http://dot.net/ml) 1.4.0-preview
+
+## **New Features**
+- **Deep Neural Networks Training (0.16.0-preview)** ([#4151](https://github.com/dotnet/machinelearning/pull/4151))  
+
+  Improves the in-preview `ImageClassification` API further:
+  - Increases DNN training speed by ~10x.
+  - Prevents repeated computations by caching featurized image values to disk from intermediate layers to train the final fully-connected layer.
+  - Reduced and constant memory footprint.
+  - Simplifies the API by not requiring the user to pre-process the image.
+  - Introduces callback to provide metrics during training such as accuracy, cross-entropy.
+  - Improved image classification sample.
+
+  ```cs
+        public static ImageClassificationEstimator ImageClassification(
+            this ModelOperationsCatalog catalog,
+            string featuresColumnName,
+            string labelColumnName,
+            string scoreColumnName = "Score",
+            string predictedLabelColumnName = "PredictedLabel",
+            Architecture arch = Architecture.InceptionV3,
+            int epoch = 100,
+            int batchSize = 10,
+            float learningRate = 0.01f,
+            ImageClassificationMetricsCallback metricsCallback = null,
+            int statisticFrequency = 1,
+            DnnFramework framework = DnnFramework.Tensorflow,
+            string modelSavePath = null,
+            string finalModelPrefix = "custom_retrained_model_based_on_",
+            IDataView validationSet = null,
+            bool testOnTrainSet = true,
+            bool reuseTrainSetBottleneckCachedValues = false,
+            bool reuseValidationSetBottleneckCachedValues = false,
+            string trainSetBottleneckCachedValuesFilePath = "trainSetBottleneckFile.csv",
+            string validationSetBottleneckCachedValuesFilePath = "validationSetBottleneckFile.csv"
+            )
+
+  ```
+
```

Comment:
Should you include that In-Memory images usage is still not supported but coming soon in next release?